addons/zeus/XEH_preInit.sqf

```diff
@@ -5,7 +5,7 @@ ADDON = false;
 #include "XEH_PREP.hpp"
 
 if (isServer) then {
-    ["ace_zeusUnitAssigned", FUNC(handleZeusUnitAssigned)] call CBA_fnc_addEventHandler;
+    [QGVAR(zeusUnitAssigned), FUNC(handleZeusUnitAssigned)] call CBA_fnc_addEventHandler;
```

Comment:
Actually, since it's added into a vanilla function, probably better to keep it private so it can be removed if needed.